### ISSUE TITLE
ScreenshotFullPage doesn't work

### ISSUE THREAD

**Issue by youshy** [ysmood/rod#50]
**Describe the bug**

Error when calling `ScreenshotFullPage` or `ScreenshotE` with `FullPage=true`

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1718b81]

goroutine 1 [running]:
github.com/ysmood/rod.(*Page).ScreenshotE(0xc001e16090, 0x18ce801, 0xc0004c9c18, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/arturkondas/go/pkg/mod/github.com/ysmood/rod@v0.35.0/page.go:293 +0x281
```

**To Reproduce**

This is the current code:
```go
body, err := page.ScreenshotE(true, &proto.PageCaptureScreenshot{})
if err != nil {
	log.Fatal(err)
}
```

**Expected behavior**
The action shouldn't fail.

**Rod Version:** v0.35.0

**OS:** Mac

**Comment by youshy** [comment]
Should [that](https://github.com/ysmood/rod/blob/master/page.go#L295):
```go
oldView := p.viewport
view := *oldView
view.Width = int64(metrics.ContentSize.Width)
view.Height = int64(metrics.ContentSize.Height)
```

be

```go
oldView := p.viewport
view := &oldView
view.Width = int64(metrics.ContentSize.Width)
view.Height = int64(metrics.ContentSize.Height)
```

**Comment by ysmood** [comment]
It's interesting that the unit test doesn't fail 😂

**Comment by ysmood** [comment]
Thanks for your report, it's just a missing init.

**Comment by ysmood** [comment]
Released `v0.35.1` with this fix
